Gems/EMotionFX/Code/EMotionFX/Source/BlendTreeAccumTransformNode.cpp

```diff
@@ -186,8 +186,11 @@ namespace EMotionFX
             default:
                 MCORE_ASSERT(false);
             }
-
-            const AZ::Quaternion targetRot = MCore::CreateFromAxisAndAngle(axis, MCore::Math::DegreesToRadians(360.0f * (inputAmount - 0.5f) * invertFactor));
+            AZ::Quaternion targetRot = AZ::Quaternion::CreateIdentity();
+            if (axis.GetLengthSq() > 0.0f)
+            {
+                targetRot = AZ::Quaternion::CreateFromAxisAngle(axis.GetNormalized(), MCore::Math::DegreesToRadians(360.0f * (inputAmount - 0.5f) * invertFactor));
```

Comment:
Nit: You can do this with a conditional operator

Reply:
can you give me an example?

Reply:
I believe so, like this

```
const AZ::Quaternion targetRotation = axis.GetLengthSq() > 0.0f
    ? AZ::Quaternion::CreateFromAxisAngle(axis.GetNormalized(), MCore::Math::DegreesToRadians(360.0f * (inputAmount - 0.5f) * invertFactor))
    : AZ::Quaternion::CreateIdentity();
```